### DISCUSSION TITLE
how to do this when using memmap_threshold

### DISCUSSION THREAD

**Discussion by raulcarlomagno** [qdrant/qdrant#3906] [General]
i wanted to try scalar quantization for my vectors
before, to prevent indexing while uploading vectors, i used to set indexing_threshold=0 before uploading, and then set it to the default value to start indexing.
i would like to try quantizacion with original vectors on disk and quantized in RAM
so i should have set this
optimizers_config=models.OptimizersConfigDiff(memmap_threshold=20000)
but, i dont know if this will try to index while uploading
should i set memmap_threshold=0 before uploading and set it to 20000 later?

thanks

**Comment by generall** [comment]
Changing `memmap_threshold` is not necessary in your case
